app/views/pages/index.html.erb

```diff
@@ -4,8 +4,8 @@
   <%= render 'parliament_dissolution_warning' %>
 
   <% if Parliament.dissolution_announced? %>
-    <a href="https://www.gov.uk/register-to-vote" class="register-to-vote">
-      Register to vote in the General Election
+    <a href="https://www.parliament.uk/get-involved/" class="get-involved">
+      Get involved with the UK Parliament
     </a>
```

Comment:
Rather than editing this we should probably switch it on a specific date in future - election day or registration closing date?